### PR TITLE
Prmp 592 - Remove the 'view record' button on the available records screen and change the details element to a normal wrapper element

### PR DESCRIPTION
**Pull request by abid-nhs**

### CODE REVIEW

File: app/src/components/generic/recordCard/RecordCard.tsx [hunk 1]
```diff
@@ -30,24 +30,19 @@ function RecordCard({
...
-                        style={{ display: 'inline-block' }}
-                        data-testid="view-record-bin"
-                    >
-                        View record
-                    </Details.Summary>
+                <div className="lloydgeorge_record-stage_pdf-expander">
                     <button
                         className="lloydgeorge_record-stage_pdf-expander-button link-button clickable"
                         data-testid="full-screen-btn"
+                        style={{ top: -30 }}
```
[the remark this answers is not in the document]
Comment (abid-nhs) [reply app/src/components/generic/recordCard/RecordCard.tsx:37]: I was wondering if inline might be better for this styling. My thought process is that I can't imagine this styling being needed or used in other places so would it be helpful putting it in an scss file?
Or is it best practice to do it in a scss file?

[the remark this answers is not in the document]
Comment (abbas-khan10) [reply app/src/components/generic/recordCard/RecordCard.tsx:37]: It's best practice not use inline in my experience, also for consistency across the codebase we have not used inline anywhere else so best hide it away in a stylesheet

[the remark this answers is not in the document]
Comment (abid-nhs) [reply app/src/components/generic/recordCard/RecordCard.tsx:37]: I'll action that. Thanks for that Abbas

File: app/src/styles/App.scss [hunk 2]
```diff
@@ -296,6 +317,9 @@ $hunit: '%';
                 margin-bottom: 0;
                 border-left: none;
                 border-right: none;
+                #full-screen-btn {
+                    top: -30px;
+                }
```
Comment (NogaNHS) [inline app/src/styles/App.scss:322]: Where is this styling coming from?

File: app/src/components/generic/recordCard/RecordCard.tsx [hunk 3]
```diff
@@ -30,24 +30,19 @@ function RecordCard({
...
-                        style={{ display: 'inline-block' }}
-                        data-testid="view-record-bin"
-                    >
-                        View record
-                    </Details.Summary>
+                <div className="lloydgeorge_record-stage_pdf-expander">
                     <button
                         className="lloydgeorge_record-stage_pdf-expander-button link-button clickable"
                         data-testid="full-screen-btn"
+                        id="full-screen-btn"
```
Comment (NogaNHS) [inline app/src/components/generic/recordCard/RecordCard.tsx:37]: We don't use 'id' selectors, can you change to 'className'?

**Comment by NogaNHS** [comment]
Can you please run 'SANDBOX - Smoke Test' on a sandbox with this branch?

File: app/src/components/generic/recordCard/RecordCard.tsx (hunk 3 above)
Comment (abid-nhs) [reply app/src/components/generic/recordCard/RecordCard.tsx:37]: Yeah sure, let me modify that now. thanks

Comment (abid-nhs) [reply app/src/components/generic/recordCard/RecordCard.tsx]: Done

File: app/src/styles/App.scss (hunk 2 above)
Comment (abid-nhs) [reply app/src/styles/App.scss]: I have removed the Detail tag wrapper as it no longer made semantical sense. But to make the "view full screen" button stay where it is, i needed to add some style to it

**Review by NogaNHS — APPROVED** [review]

**Review by bethany-kish-nhs — APPROVED** [review]

**Pull request merged**